src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,15 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
```

Comment:
I recommend doing destructuring for the object 
`const { name, email, body } = comment;`

Reply:
> I recommend doing destructuring for the object `const { name, email, body } = comment;`

Good point